CONTENT/program.markdown

```diff
@@ -3,9 +3,9 @@
 [](#english)
 # Wizardry & Magic Spells
 
-## All aspects of magic interesting to a fresh wizard.
+## Everything you need to know about our Magic sessions
```

Comment:
I mean my change was inspired by the word `Magic Sessions` because there is a difference between:
* content of magic sessions
* how each magic session is structured procedure wise ...

Maybe it could be: `... about the magic wizards learn` or something like that... is that a good compromise?